config/labwc/rc.xml

```diff
@@ -72,36 +72,12 @@
     </context>
   </mouse>
 
-  <!-- LXQt settings -->
-  <!-- space for top panel -->
-  <margin top="38"/>
-
-  <!-- positions for lxqt-runner and notifications-->
-  <regions>
-    <region name="notifications" x="70%" y="0%" height="50%" width="50%"/>
-    <region name="runner" x="40%" y="20%" height="1%" width="20%"/>
-  </regions>
   <!-- LXQt rules -->
   <windowRules>
-    <!-- For both a top and a left panel, if desired, remove MatchOnce="true".
-    Panel configuration windows will open at top left corner using this setting.
-    Other positions are not recommended -->
-    <windowRule identifier="lxqt-panel" skipTaskbar="yes" skipWindowSwitcher="yes" MatchOnce="true" fixedPosition="yes">
-      <action name="ToggleAlwaysOnBottom"/>
-      <action name="MoveTo" x="0" y="0"/>
-      <action name="ToggleKeybinds"/>
-    </windowRule>
     <windowRule identifier="lxqt-runner" skipTaskbar="yes" skipWindowSwitcher="yes">
       <action name="ToggleAlwaysOnTop"/>
       <action name="SnapToRegion" region="runner"/>
     </windowRule>
```

Comment:
You seem to have removed the `runner` region but this rule still uses it.

Reply:
It copies atm the default labwc settings to `~/.config/  if no config is present, but I've to change that again and ship a labwc folder, as default starts mako and waybar. Settingsexporter isn't working too somehow on fresh accounts I noticed yesterday. Thanks for reviewing!

Reply:
Should work now. Today LXQt 2 should be merged in git, except wayland-taskbar PR for the panel.